batch2/deployment.yaml

```diff
@@ -41,6 +41,8 @@ spec:
            value: "{{ batch2_worker_image.image }}"
          - name: BATCH_NAMESPACE
            value: "{{ default_ns.name }}"
+         - name: KUBERNETES_SERVER_URL
+           value: "{{ global.k8s_server_url }}"
```

Comment:
This needs to be set by CI.  You should PR just adding this to CI, then this will work.